frontend/assets/access.settings/res/js/AdminWorkspaces/editor/MetaNamespace.js

```diff
@@ -357,12 +379,46 @@ class MetaNamespace extends React.Component {
                     nameError={nameError}
                     styles={styles}
                 />
-                <MetaNamespaceFieldOptions 
-                    ns={namespace}
-                    ref={this.fieldOptionsRef}
-                    fieldType={JSON.parse(namespace.JsonDefinition).type}
-                />
+                {
+                    namespace?.Namespace?.toString().length > 0 && 
+                    namespace.JsonDefinition?.toString().length > 0 && 
+                    <MetaNamespaceFieldOptions 
+                        ns={namespace}
+                        ref={this.fieldOptionsRef}
+                        fieldType={JSON.parse(namespace.JsonDefinition).type}
+                    />
+                }
                 <div style={styles.section}>{Pydio.getInstance().MessageHash[310]}</div>
+                <Toggle
+                    label={'Prompt Metadata tagging on Upload'}
+                    disabled={readonly}
+                    labelPosition={"left"}
+                    toggled={namespace.PromptOnUpload ? namespace.PromptOnUpload : false}
+                    onToggle={(e, v) => { 
+                        !!namespace.PromptOnUpload ? namespace.PromptOnUpload = v : namespace.PromptOnUpload = v; this.setState({ namespace }) 
```

Comment:
This line looks unecessary
```suggestion
                        namespace.PromptOnUpload = v;
                        this.setState({ namespace });
```

Reply:
changed and rebuilt assets